src/azul/service/manifest_service.py

```diff
@@ -158,19 +178,28 @@ def get_manifest(self,
         if presigned_url is None:
             file_name = self._generate_manifest(generator, object_key)
             presigned_url = self.storage_service.get_presigned_url(object_key, file_name=file_name)
-            return presigned_url, False
+            return Manifest(location=presigned_url,
+                            was_cached=False,
+                            properties=generator.manifest_properties(presigned_url))
```

Comment:
`generator.manifest_properties(presigned_url)` is duplicated in both branches. Can't we extract a variable?

Reply:
Or, extract the Manifest instantiation into a variable. The only difference is was_cached.